internal/backends/winit/wasm_input_helper.rs

```diff
@@ -232,9 +224,7 @@ fn event_text(e: &web_sys::KeyboardEvent) -> Option<SharedString> {
         };
     }
     i_slint_common::for_each_special_keys!(check_non_printable_code);
-    if key.chars().count() == 1 {
-        Some(key.as_str().into())
-    } else {
-        None
-    }
+
+    let mut chars = key.chars();
+    chars.next().and_then(|first_char| chars.next().is_none().then(|| first_char))
```

Comment:
That's not right,  this could be a string we don't understand, such as "Finish",  then we would send "F" instead of nothing  (hence the `key.chars().count()` in the previous code.

Also, the convert function is a bit uselss now.  But maybe this should use the Key enum instead of the key_code module.

Reply:
The `chars.next().is_none()` is intended to be the replacement for `chars.count() == 1`. So the logic should be the same, if there's a character after the "F" we'll send `None`, don't we?

Reply:
I've removed the `convert` function and use explicit `Some(xxx.into())` - I think that's most readable. Not sure how to use the `Key` enum since this is the inverse mapping?

Reply:
Ah yes true, this is_none trick as not obvious when i read this line.

Reply:
Hmm, I've pushed a version with a `match` that I think is more readable. Good point though, not sure why I ended up with the less readable `then()` when it's nested.

Reply:
> removed the convert function and use explicit `Some(xxx.into())` - I think that's most readable. 

Why not just `Some(xxx)`    (i don't think you need the .into() anymore)

> Not sure how to use the Key enum since this is the inverse mapping?

`Key::Foo.into()` should work

Reply:
Ah yes, the `into()` is not needed. I was somehow hung-up on the wrong `Key` type - had forgotten that in `input` we decode the `char` costs.

I agree about using the public `Key` enum. Changed to use that (and then we still `into()`).